nucleus/constants.py

```diff
@@ -62,7 +63,8 @@
 MODEL_RUN_ID_KEY = "model_run_id"
 NAME_KEY = "name"
 NEW_ITEMS = "new_items"
-NUCLEUS_ENDPOINT = "https://api.scale.com/v1/nucleus"
+NUCLEUS_ENDPOINT = "https://jihan-yin-api.devbox.internal.scale.com/v1/nucleus"
```

Comment:
You can set this using an environment variable by the way

I have this in my bash_profile

```
alias api_local="export NUCLEUS_ENDPOINT=\"http://localhost:3000/v1/nucleus\" && echo $NUCLEUS_ENDPOINT"
alias api_prod="export NUCLEUS_ENDPOINT=\"https://api.scale.com/v1/nucleus\" && echo $NUCLEUS_ENDPOINT"
```

Reply:
sick didn't see that we grab it from the environmental variables now